Fixed harmless but annoying nullref when entering the editor

linuxgurugamer opened the pull request:
Fixed harmless but annoying nullref when entering the editor and the setup takes more than 5 seconds

linuxgurugamer committed:
Fixed harmless but annoying nullref when entering the editor and
the setup takes more than 5 seconds

ncanceill commented:
I confirm this issue, thanks @linuxgurugamer for the fix! I cannot wait for it to be merged

SumGuyTwitches committed:
kraken location off by 8 degrees

Fixed kraken location

taniwha committed:
Use dateTimeFormatter for time period lengths

This makes WaypointManager compatible with planet mods that alter
Kerbin's day length and use Kronometer.

linuxgurugamer committed:
Merge branch 'master' of git://github.com/jrossignol/WaypointManager into jrossignol-master

linuxgurugamer committed:
Merge branch 'jrossignol-master'

linuxgurugamer committed:
Merge pull request #2 from taniwha/daylength-fix

Use dateTimeFormatter for time period lengths

linuxgurugamer committed:
Merge pull request #3 from SumGuyTwitches/patch-1

kraken location off by 8 degrees

linuxgurugamer committed:
Patch from github user @SumGuyTwitches:
  fix kraken location off by 8 degrees
 Patch from Github user @taniwha:
  Use dateTimeFormatter for time period lengths

 Fixed RSS compatibility with changes listed in this issue on Github:
  https://github.com/jrossignol/WaypointManager/issues/59

linuxgurugamer committed:
Adoption by LinuxGuruGamer
 Added support for ToolbarController
 Added support for ClickThroughBlocker
 Patch from github user @SumGuyTwitches:
  fix kraken location off by 8 degrees
 Patch from Github user @taniwha:
  Use dateTimeFormatter for time period lengths

 Fixed RSS compatibility with changes listed in this issue on Github:
  https://github.com/jrossignol/WaypointManager/issues/59

linuxgurugamer committed:
Removed bad file

linuxgurugamer committed:
Added custom landing site configs, make active using PatchManager and then import into WaypointManager

linuxgurugamer committed:
Added toolbarcontroller & Clickthroughblocker support

linuxgurugamer committed:
Added page to select import of custom waypoints
 Patchmanager configs removed
 Moved icons into PluginData
 Changed loading of icons to Toolbarcontroller.LoadImageFromFile
 Added selection window to load custom waypoints
 Split big CustomWaypoints.InterestingSites.cfg into planet specific files

linuxgurugamer committed:
Fixed references

linuxgurugamer committed:
Fixed version file

linuxgurugamer committed:
Another error in version file, was https instead of http

linuxgurugamer committed:
Updated coordinates & altitude for the Armstrong Memorial

linuxgurugamer committed:
Removed extra .version file

linuxgurugamer committed:
Fixed modname on the toolbar button

linuxgurugamer committed:
disable the unfinished stock edit code

linuxgurugamer committed:
Added new dependency, SpacetuxLibrary
 Changed Debug.Log methods to Log.Info
 Added UI Scaling

linuxgurugamer committed:
Fixed loading of contract icons (a very old, unreported bug)
 Fixed scaling of text boxes to stop wraparound

linuxgurugamer committed:
Fixed loading of contract icons

linuxgurugamer committed:
Updated changelog

linuxgurugamer committed:
Added AssemblyFileVersion
 Updated version file for 1.12
 Fixed custom icons

linuxgurugamer committed:
Updated KSPAssembly

linuxgurugamer committed:
Fixed memory leak caused by the WaypointManagerGameEvents being unloaded

linuxgurugamer committed:
Fixed path for loading files (broken by new launcher)

linuxgurugamer committed:
Added ability to drag the onscreen live display of status to anywhere on screen (need to hold down either left or right control screen)
 Added reset button to settings page to reset the position of the live display to the default
 cleaned up some really old code

linuxgurugamer committed:
Fixed setting window not displaying (unintended sideeffect of the previous update)
 Added colored background (grey) for onscreen display (controllable via slider in settings window)

linuxgurugamer committed:
version file

linuxgurugamer committed:
Fixed Next flight without changing any settings, grey background is absent
     Multiple initializions are needed because KSP/Unity is stupid and loses the background texture between scenes
    Added ability to control the grey scale of the display background
 Adjusted display lines to not overlap

linuxgurugamer committed:
Thanks to github user @eminterrupted for finding this bug:

		displayOpacity was being read incorrectly from the saved config, causing the display background opacity to be reset to 50% every game restart
	Added text to config window explaining how to drag the nav window
	Updated version file to use github for version info

linuxgurugamer committed:
Updated text in Settings window re. dragging the Nav Window

	Small optimization regarding initializing styles, eliminating an extra function call during OnGUI
	Fixed nav window initialization (was not setting background/opacity upon startup)

linuxgurugamer committed:
misc, nothing major

linuxgurugamer committed:
Added option to display Waypoint Coordinates, based on code from github user @David Shade

Clayell committed:
Update CustomWaypointGUI.cs

linuxgurugamer committed:
Merge pull request #15 from Clayell/master

Edit a GUI label that is not specific to custom waypoints removing the "custom" since it's not related to custom waypoints

linuxgurugamer committed:
Thanks to github user @Clayell for this:

		Edit a GUI label that is not specific to custom waypoints removing the "custom" since it's not related to custom waypoints
	Added to bypass a Kopernicus error where it puts invalid waypoints into the system

linuxgurugamer committed:
Updated fix to not show invalid waypoints on the Sun

Clayell committed:
Update WaypointData.cs

Clayell committed:
cnt is not actually used

Clayell committed:
hide toolbar button in mainmenu

linuxgurugamer committed:
Merge pull request #16 from Clayell/master

Allow for waypoints named "Site", hide toolbar button in main menu
